Skip the logging if the pass cannot be pickled

houseroad opened the pull request:
Summary:
Skip the logging for vllm at this moment, we can add some pickle logic later.

The log is only for debugging purpose.



cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @amjames @chauhang @aakhundov

chenyang78 reviewed (approved):
lint? otherwise, LGTM. Thanks!

houseroad committed:
Skip the logging if the pass cannot be pickled (#148053)

Summary:

Skip the logging for vllm at this moment, we can add some pickle logic later.

The log is only for debugging purpose.

Reviewed By: chenyang78

Differential Revision: D70284119
